astroquery/esa/xmm_newton/core.py

```diff
@@ -406,31 +407,25 @@ def get_epic_spectra(self, filename, source_number, *,
 
                                 response = self._request('GET', link)
 
-                                rsp_filename =\
-                                    os.path.join(_path,
-                                                 paths[0],
-                                                 paths[1],
-                                                 ext.header["RESPFILE"])
+                                rsp_filename = os.path.join(_path, paths[0], paths[1], ext.header["RESPFILE"])
 
                                 with open(rsp_filename, 'wb') as f:
                                     f.write(response.content)
-                                    ret[fname_info["I"] + "_rmf"] =\
-                                        rsp_filename
+                                    ret[fname_info["I"] + "_rmf"] = rsp_filename
 
                     if ret.get(key) and type(ret.get(key)) == str:
-                        log.warning("More than one file found with the "
-                                    "instrument: %s" % key)
-                        ret[key] = [ret[key], value]
+                        log.warning("More than one file found with the instrument: %s" % key)
+                        ret[key] = [ret[key], path_inst_name]
                     elif ret.get(key) and type(ret.get(key)) == list:
-                        ret[key].append(value)
+                        ret[key].append(path_inst_name)
                     else:
-                        ret[key] = value
+                        ret[key] = path_inst_name
 
         except FileNotFoundError:
             log.error("File %s not found" % (filename))
-            return {}
+            return None
```

Comment:
`return None` is redundant; this can just be `return`

Reply:
Changed, thanks